package.json

```diff
@@ -25,7 +25,9 @@
   "scripts": {
     "build": "ember build",
     "start": "ember server",
-    "test": "ember try:each"
+    "test": "ember try:each",
+    "test:default": "ember test",
```

Comment:
I actually prefer `test: ember test` and `test:all: ember try:each` 🤔

Reply:
ya, same, I'll update to that